### ISSUE TITLE
64x64 LED Matrix w/ adafruit bonnet - black bars

### ISSUE THREAD

**Issue by dmathew93** [hzeller/rpi-rgb-led-matrix#895]
I just bought the 64x64 LED matrix and the adafruit bonnet recently and followed the instructions to assemble them appropriately. I am running into the issue where when I am running it in 64x64 mode large black bars will appear alongside 2 large blue bars. Included are the images of my setup and the display. I have done the E line from 8 to E, I have soldered the connection from 4 to 18 and tried a variety of options from the library owners github. Any help would be appreciated and would like to rule out if its the bonnet board itself.

https://imgur.com/a/kXnbrOB

**Comment by GeekyTim** [comment]
Did you solder the bonnet?

**Comment by dmathew93** [comment]
What do you mean by solder the bonnet? I soldered the E address lines and pin 4 to 18. Included an imgur link of my board.

**Comment by GeekyTim** [comment]
That would be the one.

**Comment by hzeller** [comment]
Your picture actually looks like the D line is not properly connected as all the rows with that bit set are black, so they are folded onto each other.
Try to follow the D line and see if there is any break on the circuit board or the cable.

**Comment by dmathew93** [comment]
> Your picture actually looks like the D line is not properly connected as all the rows with that bit set are black, so they are folded onto each other.
> Try to follow the D line and see if there is any break on the circuit board or the cable.

Hey Hzeller, thanks for the reply.

I just too my multimeter and started measuring the voltage on each address line. You were right. The D address line is not showing any power whatsoever. All others are showing 1.6V. That explains why the black bars are there, but i dont see how it explains the two large blue columns. Any ideas on that one?

Denzel

**Comment by dmathew93** [comment]
Trying to debug this issue with the D address line and the always on blue bars I’m now seeing this issue where the board is reacting to interference.

https://imgur.com/a/i4OSX9T

**Comment by hzeller** [comment]
when lines are a solid color typically means that they are not driven properly. This could be due to cable issues or badly soldered connections on your bonnet.
You do set the right `--led-gpio-mapping` that you need for your set-up, right ?

**Comment by dmathew93** [comment]
> when lines are a solid color typically means that they are not driven properly. This could be due to cable issues or badly soldered connections on your bonnet.
> You do set the right `--led-gpio-mapping` that you need for your set-up, right ?

Correct, I do set the mapping to the adafruit-hat one. I tried all others just to rule things out. I thought it could be the cable so I tried to measure voltages on just the board itself on the RGB pins and the B2 pin is driven to a 3V output. B1 is going to 0.

**Comment by mariana-s-fernandes** [comment]
Hi @dmathew93, please check my comment [here](https://github.com/hzeller/rpi-rgb-led-matrix/issues/1189#issuecomment-761565883) to see if that works for you.

**Comment by marcmerlin** [comment]
With apologies to everyone, this project is doing a one time cleanup of 
all the bugs that have not been addressed and closed in the last 10 years.

If you filed an issue to ask for help, please do not re-open this one or a new
bug, but go here instead: https://rpi-rgb-led-matrix.discourse.group/

Please be mindful that this project does not have any full time maintainers
or anyone paid. You should not expect personal help for your own projects or
answers to programming questions but you can ask for help with panels that
aren't working, setup questions, or anything that may be a bug here:
https://rpi-rgb-led-matrix.discourse.group/
If you need programming help and are willing to pay someone for it, please
state so in your message, and someone may be interested.

If you did report an actual bug that is still not fixed today, please accept
our apologies for this being mass closed, you can absolutely send a followup
and re-open the issue, or open a new up to date one as applicable.
Note that we very much welcome help, so if you can help fix your bug, you will
be a superstar :)

If your bug was linked to a pull request that was not resolved, please update
the pull request and the issue, and we will do our best to review them. We do
thank you for your undertanding that we have very limited free time, so any
extra work you do along with your bug report or PR request will be very much
appreciated and improve the odds that they will be looked at.

If you want to help contribute as a python or C# or any other language bindings,
you are also a superstar, please state so in the issue and we'll do our best to
give you the access and reviews you need.

For anything related to PWM panel support, please make sure you read README and
then refer to https://github.com/hzeller/rpi-rgb-led-matrix/issues/466 . 
Please do not comment if you're haven't read everything and/or if you're just
asking when your panel is going to be supported. Those bugs are if you're 
able to contribute to the effort.

For rPI5: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1603

For anything python, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
For anything C#, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
